python/lsst/ts/standardscripts/auxtel/calibrations/power_off_atcalsys.py

```diff
@@ -125,9 +126,7 @@ async def wait_for_lamp_to_cool_down(self):
                 lamp_state = await self.white_light_source.evt_lampState.next(
                     flush=False, timeout=self.timeout_lamp_cool_down
                 )
-                cool_down_wait_time = (
-                    lamp_state.cooldownEndTime - lamp_state.private_sndStamp
-                )
+                cool_down_wait_time = lamp_state.cooldownEndTime - time.time()
```

Comment:
I think what you want to do is replace `cool_down_wait_time` to `self.timeout_lamp_cool_down` in line 132. This here doesn't really make sense.  Or, actually, just remove line 132.

Reply:
Removed line 132 and caught a related bug lower down in the exception message.